src/Workspaces/CSharp/Portable/Recommendations/CSharpRecommendationServiceRunner.cs

```diff
@@ -491,16 +491,16 @@ private RecommendedSymbols GetSymbolsOffOfConditionalReceiver(ExpressionSyntax o
             // applicable to it as well.
             var unnamedSymbols = _context.IsNameOfContext || excludeInstance
                 ? default
-                : GetUnnamedSymbols(originalExpression);
+                : GetUnnamedSymbols(containerSymbol, originalExpression);
```

Comment:
`containerSymbol` was found by speculation and needs to be passed. Here the speculation takes place:

https://github.com/dotnet/roslyn/blob/4ab21f1b9b6125c783531ab7a0d2479efd5fe306/src/Workspaces/CSharp/Portable/Recommendations/CSharpRecommendationServiceRunner.cs#L280-L284

This is the explanation given there:
```CS
            // Check if we're in an interesting situation like this:
            //
            //     int i = 5;
            //     i.          // <-- here
            //     List<string> ml = new List<string>();
            //
            // The problem is that "i.List<string>" gets parsed as a type.  In this case we need 
            // to try binding again as if "i" is an expression and not a type.  In order to do 
            // that, we need to speculate as to what 'i' meant if it wasn't part of a local 
            // declaration's type.
            //
            // Another interesting case is something like:
            //
            //      stringList.
            //      await Test2();
            //
            // Here "stringList.await" is thought of as the return type of a local function.
```